### DISCUSSION TITLE
🔗 Include/Exclude taxonomies from the Taxonomy Filter (Related to #23418)

### DISCUSSION THREAD

**Discussion by awakekat** [elementor/elementor#23185] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When adding a Loop Grid widget to the page, I am able to go into the query for the Loop Grid and exclude a Term Category. When I add the new Taxonomy Filter above the Loop Grid Widget, I am unable to replicate the exclusion of the Term Category in the Taxonomy Filter from the Loop Grid I previously made.  The feature of excluding certain term categories from the Taxonomy Filter does not exist at this time.

### Describe the solution you'd like

It would be great if I was able to replicate the ability to exclude certain term categories from the Taxonomy filter in order to replicate the same categories being listed in the Loop Grid. For example if I have 3 categories: News, Events, and Q&A - I want to exclude the Q&A to a different page - I can exclude Q&A in the custom loop grid, but cannot in the taxonomy filter - I still get all three categories in the taxonomy filter.

**Comment by yyc-food-blog** [comment]
Second this. An alternative solution is that when "hide empty taxonomies" is selected, it hides empty taxonomies for the query in the loop. So if the query isn't pulling anything from a taxonomy, then it doesn't show in the filter list.

**Comment by getwebcare** [comment]
+1 
I have News and different Portfolio categories in posts and I cannot exclude News from showing in my Portfolio loop.

Edit.
Currently using a workaround with css to hide the News category from filter bar

`.e-filter-item[data-filter="News"] {
    display: none!important;
  }`

**Comment by nicholaszein** [comment]
Hello @awakekat,

Thanks for contributing your suggestion! 🙏

Your ideas and input as a Web Creator are valuable to us. We're always keen to explore ways to enhance Elementor to meet our community's needs better.

### 💡 Due to the high volume of feature requests we receive and our commitment to continuous improvement, we can't guarantee that every suggestion will be immediately included in our roadmap.

We appreciate your patience as we take the time to review your request thoroughly. We aim to ensure all decisions align with our mission to provide Web Creators with the tools they need to build their dream websites.

Kind regards

**Comment by nicholaszein** [comment]
This feature is related to, or mentioned in the following Issues and Discussions:

- #23418
- #24631

**Comment by davidamarek** [comment]
+1, would be great to exclude certain categories from the filter.

**Comment by saawood13** [comment]
Need this feature!!

**Comment by saawood13** [comment]
Really frustrating that we cannot have a query added to a loop grid as well as using a taxonomy filter with a different query. I have two post categories 'Blogs' and 'Varieties' and used a query on the loop grid to show only the 'variety' post categories. But within all the 'variety' posts, they have unique tags. I want a taxonomy filter that shows tags while the loop grid only shows posts within the category 'varieties'

**Comment by heroicP** [comment]
+1 I can't believe this isn't already a feature. Who wants to show ALL of their tags in a filter menu?!

**Comment by kskonovalov** [comment]
Absolutely must needed, please add!
